src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -3940,7 +3984,7 @@ private void VisitThisOrBaseReference(BoundExpression node)
         public override BoundNode VisitParameter(BoundParameter node)
         {
             var parameter = node.ParameterSymbol;
-            int slot = GetOrCreateSlot(parameter);
+            int slot = GetOrCreateSlot(parameter, createIfMissing: false);
```

Comment:
>createIfMissing: false [](start = 50, length = 22)

I am not sure if passing this value is going to have any useful effect. If we are not interested in allocating new slots, perhaps we should use VariableSlot API instead.